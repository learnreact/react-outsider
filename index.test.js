import React from "react"
import Outsider from "./"
import renderer from "react-test-renderer"

test("tests work", () => {
  const component = renderer.create(<Outsider />)

  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
