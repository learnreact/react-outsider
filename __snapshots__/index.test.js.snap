// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests work 1`] = `<div />`;
